publish/publish.proj

```diff
@@ -134,14 +134,17 @@
   </Target>
 
   <Target Name="PublishCoreHostPackagesToFeed"
```

Comment:
There's several different "Publish" things going on in this file - publishing packages to Azure, publishing packages to NuGet, publishing .deb packages to their repo, publish debian tools to NuGet.  I think we should continue to be specific here or it could get confusing.  We could rename `PushCoreHostPackagesToFeed` to `PushCoreHostPackagesToNuGetFeed` maybe?